plugins/hls-pragmas-plugin/test/Main.hs

```diff
@@ -176,8 +170,8 @@ mkEdit :: (UInt,UInt) -> (UInt,UInt) -> T.Text -> TextEdit
 mkEdit (startLine, startCol) (endLine, endCol) newText =
     TextEdit (Range (Position startLine startCol) (Position endLine endCol)) newText
 
-completionTest :: String -> FilePath -> T.Text -> T.Text -> Maybe InsertTextFormat -> Maybe T.Text -> Maybe T.Text -> [UInt] -> TestTree
-completionTest testComment fileName replacementText expectedLabel expectedFormat expectedInsertText detail [delFromLine, delFromCol, delToLine, delToCol, completeAtLine, completeAtCol] =
+completionTest :: String -> FilePath -> T.Text -> T.Text -> Maybe InsertTextFormat -> Maybe T.Text -> Maybe T.Text -> (UInt, UInt, UInt, UInt, UInt, UInt) -> TestTree
```

Comment:
I find it less evil to use 6-tuple than to disable incomplete patterns warning just to be able to pass fixed length list :smirk: